aleph-client/src/utility.rs

```diff
@@ -6,17 +6,33 @@ use crate::{
     BlockHash, Connection,
 };
 
+/// Any object that implements retrieving various info about blocks.
 #[async_trait::async_trait]
 pub trait BlocksApi {
+    /// Returns the first block of a session.
+    /// * `session` - number of the session to query the first block from
     async fn first_block_of_session(&self, session: SessionIndex) -> Option<BlockHash>;
+
+    /// Returns hash of a given block.
+    /// * `block` - number of the block
     async fn get_block_hash(&self, block: BlockNumber) -> Option<BlockHash>;
```

Comment:
Returns block to smt like Returns hash of block if given block exists

Reply:
done